src/twisted/application/twist/test/test_twist.py

```diff
@@ -157,18 +138,27 @@ def test_run(self):
             Runner, "run", lambda self: None
         )
 
-        twistOptions = Twist.options(["twist", "web"])
-        runnerArguments = Twist.runnerArguments(twistOptions)
-        Twist.run(runnerArguments)
+        twistOptions = Twist.options([
```

Comment:
I do have to say that seeing these little classmethod invocations gives me a good feeling about where this API is going; it seems quite expressive.